tests/pages/AddIdentityInputForm.test.tsx

```diff
@@ -52,7 +52,9 @@ describe("AddNewIdentity", () => {
     expect(addIdentityInput).toHaveLength(1);
   });
 
-  test("it should submit the form", () => {
+  test("it should call `send-identity-validation-code` API page on submit", () => {
+    Date.now = jest.fn(() => 1572393600000);
```

Comment:
```suggestion
    const spyDate = jest.spyOn(Date, "now").mockReturnValue(1572393600000);
```

with:

```
spyDate.mockRestore()
```

at the ned of the test